__tests__/app.test.js

```diff
@@ -470,3 +470,30 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe.only("DELETE /api/comments/:comment_id", () => {
+  test("should respond with a status 204", () => {
+    return request(app)
+      .delete("/api/comments/1")
+      .expect(204)
+      .then((response) => {
+        expect(response._body).toBe(undefined);
```

Comment:
No need to check for the response body, checking that it returns status code 204 is enough :)